### PR TITLE
Fixes #1036

### LINKED ISSUE #1036: Quickstart / Accumulo 2.x deployment is broken on master branch

### PR DESCRIPTION
**Pull request by keith-ratcliffe**
- Updated Accumulo from 2.0 to 2.0.1
 - Fixed ClassCastExceptions thrown in Pushdown[Function|Scheduler]
 - Fixed ClassNotFoundException during webservice EAR deployment
 - Fixed Tserver FileNotFoundException on TypeMetadataBridgeContext.xml
 - Fixed CLASSPATH setup errors in some ingest scripts

### CODE REVIEW

**Review by apmoriarty — APPROVED** [review]

**Comment by keith-ratcliffe** [comment]
Rebased and force pushed an update, to address junit failures due to ClassCastException in PushdownFunction...casting InMemoryAccumuloClient to WrappedAccumuloClient (introduced by my first commit)

**Comment by keith-ratcliffe** [comment]
I believe my last commit concludes work on #1036, pending review of course

**Review by apmoriarty — DISMISSED** [review]

File: warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownFunction.java [hunk 1]
```diff
@@ -175,9 +173,13 @@ protected void redistributeQueries(Multimap<String,QueryPlan> serverPlan, Tablet
...
             binnedRanges.clear();
-            AuthenticationToken authToken = new PasswordToken(config.getAccumuloPassword());
-            Credentials creds = new Credentials(config.getClient().whoami(), authToken);
-            List<Range> failures = tl.binRanges((ClientContext) client, ranges, binnedRanges);
+            ClientContext ctx;
+            if (client instanceof WrappedAccumuloClient) {
+                ctx = (ClientContext) ((WrappedAccumuloClient) client).getReal();
+            } else {
+                ctx = (ClientContext) client;
+            }
```
Comment (brianloss) [inline warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownFunction.java:181]: I think there's a lot of this in the code. Would probably be a good idea to move this into a static utility method somewhere.

Comment (keith-ratcliffe) [reply warehouse/query-core/src/main/java/datawave/query/scheduler/PushdownFunction.java]: Moved to static utility method  in latest commit

**Review by brianloss — APPROVED** [review]
LGTM

**Review by ivakegg — APPROVED** [review]

**Pull request merged**